tests/helpers.py

```diff
@@ -58,14 +58,32 @@ def search(self):
         pass
 
 
-def call_script(args_list, delete_test_reports=True):
+def delete_report_files(file_extension):
+    """
+    Delete all files with the given extension from the test_reports directory.
+
+    Args:
+        file_extension: All files with this extension will be deleted
+    """
+    try:
+        filelist = [f for f in os.listdir('test_reports') if f.endswith(file_extension)]
```

Comment:
Nit: Any reason for this line to be in the `try...except` block instead of outside it?

Reply:
Nope, I'll move it.